ext/QuantumToolboxCUDAExt.jl

```diff
@@ -10,56 +10,64 @@ import SparseArrays: SparseVector, SparseMatrixCSC
 
 If `A.data` is a dense array, return a new [`QuantumObject`](@ref) where `A.data` is in the type of `CUDA.CuArray` for gpu calculations.
 """
-CuArray(A::QuantumObject{Tq}) where Tq<:Union{Vector,Matrix} = QuantumObject(CuArray(A.data), A.type, A.dims)
+CuArray(A::QuantumObject{Tq}) where {Tq<:Union{Vector,Matrix}} =
+    QuantumObject(CuArray(A.data), A.type, A.dims)
 
 @doc raw"""
     CuArray{T}(A::QuantumObject)
 
 If `A.data` is a dense array, return a new [`QuantumObject`](@ref) where `A.data` is in the type of `CUDA.CuArray` with element type `T` for gpu calculations.
 """
-CuArray{T}(A::QuantumObject{Tq}) where {T,Tq<:Union{Vector,Matrix}} = QuantumObject(CuArray{T}(A.data), A.type, A.dims)
+CuArray{T}(A::QuantumObject{Tq}) where {T,Tq<:Union{Vector,Matrix}} =
+    QuantumObject(CuArray{T}(A.data), A.type, A.dims)
```

Comment:
Should we increase the `margin=120` (`default:92`)?
Since our type names (e.g., `QuantumObject{<:AbstractArray{T}}`, `SuperOperatorQuantumObject`, and so on) is quite long. Many single line functions will become two lines.

The case I tagged here has (original) `length=113`.